pkgs/by-name/de/deno/package.nix

```
@@ -21,25 +22,33 @@
   nodejs,
   git,
   python3,
+  esbuild,
 }:
 
 let
   canExecute = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
 in
 rustPlatform.buildRustPackage (finalAttrs: {
   pname = "deno";
-  version = "2.3.5";
+  version = "2.3.6";
 
   src = fetchFromGitHub {
     owner = "denoland";
     repo = "deno";
     tag = "v${finalAttrs.version}";
     fetchSubmodules = true; # required for tests
-    hash = "sha256-lu3r1v3iB9NIruooRrV9NawUnKqufqlYJQe+Aumgn8E=";
+    hash = "sha256-l3cWnv2cEmoeecYj38eMIlgqlRjDbtQuc6Q3DmOJoqE=";
   };
 
   useFetchCargoVendor = true;
-  cargoHash = "sha256-XJy7+cARYEX8tAPXLHJnEwXyZIwPaqhM7ZUzoem1Wo0=";
+  cargoHash = "sha256-UTnT+spuP9EJmvDDzDLJ4pxbIJhryzwBEU9SQi1UbVM=";
+  cargoPatches = [
+    (fetchpatch {
+      name = "fix-sigsegv-on-x86_64-unknown-linux-gnu-targets";
+      url = "https://github.com/denoland/deno/commit/9dd0e9aa15dfab867c16155aa719a2ed38b23308.patch";
+      hash = "sha256-gf0seGjeS7yyXIRZq/XMKqhtvXkv1+TufCBcCudv+gA=";
+    })
```

```suggestion
    (fetchpatch {
      name = "fix-sigsegv-on-x86_64-unknown-linux-gnu-targets";
      url = "https://github.com/denoland/deno/commit/400a9565c335b51d78c8909f4dbf1dbd4fb5e5d8.patch";
      hash = "";
      includes = [ "cli/Cargo.toml" ];
    })
```

The original PR was reverted in https://github.com/denoland/deno/pull/29806

---

Ah, this can be applied in `patches` instead of `cargoPatches` now, by the way, because there are no changes to `Cargo.lock`.